osfoffline/tasks/operations.py

```diff
@@ -121,7 +121,6 @@ class LocalCreateFile(BaseOperation):
     """Download an individual file from the OSF into a folder that already exists"""
 
     def _run(self):
-        logger.info('LocalCreateFile: {}'.format(self.remote))
```

Comment:
Agreed! We were actually logging this info more than once, and I was trying to consolidate it- putting the message in the execution of the job queue instead of in the job method itself. Let me know if you'd prefer it be here instead?

https://github.com/CenterForOpenScience/OSF-Offline/pull/116/files#diff-ee86bf30e5e21f98e3c70a670a827861R32